lua/terrortown/menus/gamemode/thirdparty/thirdparty.lua

```diff
@@ -0,0 +1,18 @@
+--- @ignore
+
+CLGAMEMODESUBMENU.base = "base_gamemodesubmenu"
+
+CLGAMEMODESUBMENU.priority = 100
+CLGAMEMODESUBMENU.title = "submenu_thirdparty_general_title"
+
+function CLGAMEMODESUBMENU:Populate(parent)
+	local form = vgui.CreateTTT2Form(parent, "header_thirdparty_general")
+
+	form:MakeHelp({
+		label = "help_thirdparty_enable_workshop"
+	})
+	form:MakeCheckBox({
+		serverConvar = "ttt2_thirdparty_enable_workshop",
```

Comment:
this does not exist anymore (at least i cannot find it and also i think it should be removed).

Also if we remove this, i think the whole third-party menu can be remvoed aswell.